protocol/daemons/slinky/client/client_test.go

```diff
@@ -73,7 +73,7 @@ func TestClient(t *testing.T) {
 	)
 	waitTime := time.Second * 5
 	require.Eventually(t, func() bool {
-		return cli.HealthCheck() == nil
+		return cli.GetMarketPairHC().HealthCheck() == nil || cli.GetPriceHC().HealthCheck() == nil
```

Comment:
since we want to wait for both to be successful, shouldn't this be &&?
```suggestion
		return cli.GetMarketPairHC().HealthCheck() == nil && cli.GetPriceHC().HealthCheck() == nil
```